source/2_universal_payload.rst

```diff
@@ -3,6 +3,10 @@
 Universal Payload
 ===================
 
+July 30, 2024
+
```

Comment:
i think this would be enough? no need the index.html :)
 https://universalpayload.github.io/spec/